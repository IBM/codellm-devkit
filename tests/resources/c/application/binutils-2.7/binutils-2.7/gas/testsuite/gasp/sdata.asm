

	.SDATA	"HI","STEVE"
	.SDATA	"HI" , "STEVE" , <72>,<73>,<83><69><86><69>

	.SDATA	"H""I" , "STEVE" , <72>,<73>,<83><69><86><69>



	.SDATA	"SHOULD NOT FAIL" "HERE" 
	.SDATA	"SHOULD FAIL"  foo "HERE" 

	.SDATAB	8,"BOINK"

	; examples from book

	.SDATAB	2,"AAAAA"
	.SDATAB	2,"""BBB"""
	.SDATAB	2,"AABB"<H'07>


a1:	.SDATAZ	"HI"
a2:	.SDATAC "HI"
a3:	.SDATA	"HI"
