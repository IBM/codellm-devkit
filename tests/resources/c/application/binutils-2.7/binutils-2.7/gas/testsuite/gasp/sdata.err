sdata.asm:19 Character code in string must be absolute expression.
sdata.asm:19 Missing > for character code.
END missing from end of file.
