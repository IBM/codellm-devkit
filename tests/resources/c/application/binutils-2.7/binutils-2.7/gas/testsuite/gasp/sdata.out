!

!

!	.SDATA	"HI","STEVE"
	.byte	72,73,83,84,69,86,69
!	.SDATA	"HI" , "STEVE" , <72>,<73>,<83><69><86><69>
	.byte	72,73,83,84,69,86,69,72,73,83,69,86,69
!

!	.SDATA	"H""I" , "STEVE" , <72>,<73>,<83><69><86><69>
	.byte	72,34,73,83,84,69,86,69,72,73,83,69,86,69
!

!

!

!	.SDATA	"SHOULD NOT FAIL" "HERE" 
	.byte	83,72,79,85,76,68,32,78,79,84,32,70,65,73,76,72,69,82,69
!	.SDATA	"SHOULD FAIL"  foo "HERE" 
	.byte	83,72,79,85,76,68,32,70,65,73,76,102,111,111,32,34,72,69,82,69,34,32
!

!	.SDATAB	8,"BOINK"
	.byte	66,79,73,78,75
	.byte	66,79,73,78,75
	.byte	66,79,73,78,75
	.byte	66,79,73,78,75
	.byte	66,79,73,78,75
	.byte	66,79,73,78,75
	.byte	66,79,73,78,75
	.byte	66,79,73,78,75
!

!	; examples from book
	; examples from book
!

!	.SDATAB	2,"AAAAA"
	.byte	65,65,65,65,65
	.byte	65,65,65,65,65
!	.SDATAB	2,"""BBB"""
	.byte	34,66,66,66,34
	.byte	34,66,66,66,34
!	.SDATAB	2,"AABB"<H'07>
	.byte	65,65,66,66,0
	.byte	65,65,66,66,0
!

!

!a1:	.SDATAZ	"HI"
a1:	.byte	72,73,0
!a2:	.SDATAC "HI"
a2:	.byte	2,72,73
!a3:	.SDATA	"HI"
a3:	.byte	72,73
!